mongodb/migrations/migrations_test.go

```diff
@@ -183,6 +183,176 @@ func TestMigrations_m004(t *testing.T) {
 }
 
+func TestMigrations_m005(t *testing.T) {
+	ctx := context.Background()
+	db := setup(t, ctx)
+	_, err := db.Collection("bucketarchives").InsertMany(ctx, []interface{}{
+		bson.M{
+			"_id": "id1",
+			"archives": bson.M{
+				"current": bson.M{
+					"job_id":     "job3",
+					"status":     1,
+					"created_at": 1000,
+				},
+				"history": bson.A{
+					bson.M{
+						"job_id":     "job2",
+						"status":     1,
+						"created_at": 1000,
+					},
+					bson.M{
+						"job_id":     "job1",
+						"status":     1,
+						"created_at": 1000,
+					},
+				},
+			},
+		},
+		bson.M{
+			"_id": "id2",
+			"archives": bson.M{
+				"current": bson.M{
+					"job_id":     "job6",
+					"job_status": 1,
+					"created_at": 1000,
+				},
+				"history": bson.A{
+					bson.M{
+						"job_id":     "job5",
+						"job_status": 1,
+						"created_at": 1000,
+					},
+					bson.M{
+						"job_id":     "job4",
+						"job_status": 1,
+						"created_at": 1000,
+					},
+				},
+			},
+		},
+		bson.M{
+			"_id": "id3",
+			"archives": bson.M{
+				"current": bson.M{
+					"job_id":     "job9",
+					"status":     1,
+					"created_at": 1000,
+				},
+				"history": bson.A{
+					bson.M{
+						"job_id":     "job8",
+						"job_status": 1,
+						"created_at": 1000,
+					},
+					bson.M{
+						"job_id":     "job7",
+						"status":     1,
+						"created_at": 1000,
+					},
+				},
+			},
+		},
+		bson.M{
+			"_id": "id4",
+			"archives": bson.M{
+				"current": bson.M{
+					"job_id":     "job12",
+					"job_status": 1,
+					"created_at": 1000,
+				},
+				"history": bson.A{
+					bson.M{
+						"job_id":     "job11",
+						"status":     1,
+						"created_at": 1000,
+					},
+					bson.M{
+						"job_id":     "job10",
+						"status":     1,
+						"created_at": 1000,
+					},
+				},
+			},
+		},
+		bson.M{
+			"_id": "id5",
+			"archives": bson.M{
+				"history": bson.A{
+					bson.M{
+						"job_id":     "job14",
+						"job_status": 1,
+						"created_at": 1000,
+					},
+					bson.M{
+						"job_id":     "job13",
+						"status":     1,
+						"created_at": 1000,
+					},
+				},
+			},
+		},
+		bson.M{
+			"_id": "id6",
+			"archives": bson.M{
+				"current": bson.M{
+					"job_id":     "job15",
+					"job_status": 1,
+					"created_at": 1000,
+				},
+			},
+		},
+	})
+	require.NoError(t, err)
+
+	// Run up
+	err = migrate.NewMigrate(db, m005).Up(migrate.AllAvailable)
+	require.NoError(t, err)
+
+	count, err := db.Collection("bucketarchives").CountDocuments(ctx, bson.M{})
+	require.NoError(t, err)
+	assert.Equal(t, 6, int(count))
+	cursor, err := db.Collection("bucketarchives").Find(ctx, bson.M{})
+	require.NoError(t, err)
+	defer cursor.Close(ctx)
+
+	validateItem := func(item bson.M) {
+		require.NotEmpty(t, item["job_id"])
+		require.Contains(t, item["job_id"], "job")
+		require.Equal(t, 1000, int(item["created_at"].(int32)))
+		require.Nil(t, item["job_status"])
+		require.Equal(t, 1, int(item["status"].(int32)))
+	}
```

Comment:
Little func that validates a bucket archive object. Most important thing is to make sure the id still looks valid, `job_status` no longer exists and `status` does exist.